docs/install/install_instructions.md

```diff
@@ -360,7 +360,7 @@ Particularly useful CMake options are (use **ON** to enable and **OFF** to disab
 * **-DCMAKE_INSTALL_PREFIX=/install/dir/path** : Location for installing
 * **-DCORENRN\_ENABLE\_NMODL=ON** : Use [NMODL](https://github.com/BlueBrain/nmodl/) instead of [MOD2C](https://github.com/BlueBrain/mod2c/) for code generation with CoreNEURON
 
-Please refer to [docs/cmake_doc/options.rst](docs/cmake_doc/options.rst) for more information on
+Please refer to [docs/cmake_doc/options.rst](https://github.com/neuronsimulator/nrn/blob/master/docs/cmake_doc/options.rst) for more information on
```

Comment:
I believe we are using a relative path than an absolute URL for local or CI docs build. But I see that it's non-existent one today: https://nrn.readthedocs.io/en/latest/install/docs/cmake_doc/options.rst

I will let @alexsavulescu to take a look.

Reply:
Thanks @cccohen for pointing this out, I've been able to find a general solution via #2322

Reply:
You're welcome! I hope this did not cause unnecessary work for you.